### ISSUE TITLE
Php 8 supports

### ISSUE THREAD

**Issue by ghost** [setiawanhu/laravel-madeline-proto#8]
**Describe the bug**

**To Reproduce**
1. Go to composer install
```
  Problem 2
    - setiawanhu/laravel-madeline-proto 1.1.0 requires php ^7.4 -> your PHP version (8.0.0) does not satisfy that requirement.
    - setiawanhu/laravel-madeline-proto 1.1.0 requires php ^7.4 -> your PHP version (8.0.0) does not satisfy that requirement.
    - Installation request for setiawanhu/laravel-madeline-proto ^1.1 -> satisfiable by setiawanhu/laravel-madeline-proto[1.1.0].
```
**Expected behavior**
can install on php 8.0

**Comment by setiawanhu** [comment]
Thanks for your feedback!

**Comment by setiawanhu** [comment]
Fixed at #9
